pkg/util/validation/limits.go

```diff
@@ -243,6 +244,7 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	f.IntVar(&l.MaxLabelNamesPerSeries, MaxLabelNamesPerSeriesFlag, 30, "Maximum number of label names per series.")
 	f.IntVar(&l.MaxMetadataLength, MaxMetadataLengthFlag, 1024, "Maximum length accepted for metric metadata. Metadata refers to Metric Name, HELP and UNIT. Longer metadata is dropped except for HELP which is truncated.")
 	f.IntVar(&l.MaxNativeHistogramBuckets, maxNativeHistogramBucketsFlag, 0, "Maximum number of buckets per native histogram sample. 0 to disable the limit.")
+	f.IntVar(&l.MaxExemplarsPerSeriesPerRequest, "distributor.max-exemplars-per-series-per-request", 100, "Maximum ratio of exemplars to series per request. 0 to disable limit in request. The exceeding exemplars are dropped.")
```

Comment:
Why not follow the example of `validation.max-label-names-per-series`? The `-per-request` suffix seems overly verbose:
```suggestion
	f.IntVar(&l.MaxExemplarsPerSeriesPerRequest, "distributor.max-exemplars-per-series", 100, "Maximum ratio of exemplars to series per request. 0 to disable limit in request. The exceeding exemplars are dropped.")
```

I also wonder if the prefix should be `validation.`, as for `validation.max-label-names-per-series`. @pstibrany WDYT?

Reply:
I had doubt on validation or distributor, some of the limits check are in distributor and some are in validation. I would also want a second opinion on our rules.

Reply:
We have flag names with both `distributor` and validation prefix and no good rule what prefix to use. Ideally CLI flag name would reflect YAML path in the config, which in this case would be `limits.max_exemplars_per_series_per_request`. :) I'd keep `distributor.`, but don't have a strong preference.